Packs/Exterro/Integrations/Exterro/Exterro.py

```diff
@@ -0,0 +1,112 @@
+"""Base Integration for Cortex XSOAR (aka Demisto)
+
+This is an empty Integration with some basic structure according
+to the code conventions.
+
+MAKE SURE YOU REVIEW/REPLACE ALL THE COMMENTS MARKED AS "TODO"
+
+Developer Documentation: https://xsoar.pan.dev/docs/welcome
+Code Conventions: https://xsoar.pan.dev/docs/integrations/code-conventions
+Linting: https://xsoar.pan.dev/docs/integrations/linting
+
+This is an empty structure file. Check an example at;
+https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py
+
+"""
+
+# python 3.9 imports
+from functools import wraps
+from json import dumps, loads, JSONDecodeError
+from traceback import format_exc
+
+# accessdata imports
+from accessdata.client import Client
+
+# xsoar imports
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+""" decorator wrapping demisto commands """
+
+_run_functions = {}
+
+
+def wrap_demisto_command(command):
+    def _func(func):
+        @wraps(func)
+        def _inside(*args, **kwargs):
+            return func(*args, **kwargs)
+
+        _run_functions[command] = func
+        return _inside
+
+    return _func
```

Comment:
While this is a valid code, note that it will be harder for you to debug the different commands and is not recommended.